src/index.html

```diff
@@ -2,11 +2,450 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="value">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a 
+        href="#home"
+        class="header-logo"
+      >
+        <img
+          src="/images/logo.png" 
+          alt="#logo"
+        >
+      </a>
+      <nav class="header-nav">
```

Comment:
```suggestion
      <nav class="nav">
```
це окремий бем блок має бути